tests/sentry/feedback/test_feedback_ingest.py

```diff
@@ -0,0 +1,180 @@
+
+test_data = {
+    "contexts": {},
+    "tags": {
+        "sentry_version": "23.9.0.dev0",
+    },
+    "timestamp": 1694039635.9195,
+    "message": "This website is great!",
+    "transaction": "/replays/",
+    "type": "transaction",
+    "transaction_info": {"source": "route"},
+    "platform": "javascript",
+    "event_id": "b51647a3c56f4a939984bb1147a6c3e5",
+    "environment": "prod",
+    "release": "frontend@40f88cd929122ac73749cc48f0ddb9aa223449ff",
+    "sdk": {"name": "sentry.javascript.react", "version": "7.66.0-alpha.0"},
+    "user": {
+        "ip_address": "72.164.175.154",
+        "email": "josh.ferge@sentry.io",
+        "id": 880461,
+        "isStaff": False,
+        "name": "Josh Ferge",
+    },
+    "request": {
+        "url": "https://sentry.sentry.io/replays/?project=11276&statsPeriod=7d",
+        "headers": {
+            "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36"
+        },
+    },
+}
+
+test_data_missing_optional_fields = {
+    "timestamp": 1694039635.9195,
+    "message": "This website is great!",
+    "transaction": "/replays/",
+    "type": "transaction",
+    "transaction_info": {"source": "route"},
+    "platform": "javascript",
+    "event_id": "b51647a3c56f4a939984bb1147a6c3e5",
+    "request": {
+        "url": "https://sentry.sentry.io/replays/?project=11276&statsPeriod=7d",
+        "headers": {
+            "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36"
+        },
+    },
+}
+
+wrong_test_data = {
+    "contexts33": {},
+    "tags!": {
+        "sentry_version": "23.9.0.dev0",
+    },
+    "platform_bad": "javascript",
+    "event_id": "b51647a3c56f4a939984bb1147a6c3e5",
+}
+
+missing_event_id_test_data = {
+    "contexts": {},
+    "tags": {
+        "sentry_version": "23.9.0.dev0",
+    },
+    "timestamp": 1694039635.9195,
+    "message": "This website is great!",
+    "transaction": "/replays/",
+    "type": "transaction",
+    "transaction_info": {"source": "route"},
+    "platform": "javascript",
+    "environment": "prod",
+    "release": "frontend@40f88cd929122ac73749cc48f0ddb9aa223449ff",
+    "sdk": {"name": "sentry.javascript.react", "version": "7.66.0-alpha.0"},
+    "user": {
+        "ip_address": "72.164.175.154",
+        "email": "josh.ferge@sentry.io",
+        "id": 880461,
+        "isStaff": False,
+        "name": "Josh Ferge",
+    },
+    "request": {
+        "url": "https://sentry.sentry.io/replays/?project=11276&statsPeriod=7d",
+        "headers": {
+            "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36"
+        },
+    },
+}
+
+wrong_type_test_data = {
+    "timestamp": "1694039635.9195",
+    "message": 24,
+    "event_id": "b51647a3c56f4a939984bb1147a6c3e5",
+    "transaction": {},
+}
```

Comment:
If these are one-off payloads then they can be moved inside the test itself.  It makes it easier to see what's being passed.